Lib/unittest/case.py

```diff
@@ -1329,6 +1329,13 @@ def assertNotRegex(self, text, unexpected_regex, msg=None):
             msg = self._formatMessage(msg, standardMsg)
             raise self.failureException(msg)
 
+    def _runTestMethod(self, method):
```

Comment:
Since this is documented as something that can be overloaded in subclasses, I think it should be a public API.

Reply:
+1

Reply:
Done!

Should it be publicly documented (in unittest.rst) as well?